samples/ballerina-by-example/examples/websocket-basic-functions/websocket-basic-functions.bal

```diff
@@ -0,0 +1,29 @@
+import ballerina.lang.system;
+import ballerina.lang.messages;
+import ballerina.net.http;
+import ballerina.net.ws;
+import ballerina.doc;
+
+@http:BasePath {value:"/endpoint"}
+@ws:WebSocketUpgradePath {value:"/ws"}
+service websocketEndpoint {
+
+    @ws:OnOpen {}
+    resource onOpen(message m) {
+        // broadcast text to all connected clients
+        ws:broadcastText("New client connected");
+    }
+
+    @ws:OnTextMessage {}
+    resource onTextMessage(message m) {
+        // Push text to the same client who sent the message
+        ws:pushText("You said: " + messages:getStringPayload(m));
```

Comment:
This line goes beyond the source code box when generated the HTML page using build tool.